plugins/package-managers/node/src/funTest/assets/projects/synthetic/yarn/workspaces-expected-output.yml

```diff
@@ -15,14 +15,52 @@ project:
     type: "Git"
     url: "<REPLACE_URL_PROCESSED>"
     revision: "<REPLACE_REVISION>"
-    path: "plugins/package-managers/node/src/funTest/assets/projects/synthetic/yarn/workspaces"
+    path: "plugins/package-managers/node/src/funTest/assets/projects/synthetic/yarn/workspaces/packages/pkg2"
   homepage_url: ""
   scopes:
   - name: "dependencies"
     dependencies:
     - id: "NPM:@here:harp-fetch:0.3.6"
       dependencies:
       - id: "NPM::node-fetch:2.6.0"
+- id: "Yarn::workspace:1.0.0"
+  definition_file_path: "plugins/package-managers/node/src/funTest/assets/projects/synthetic/yarn/workspaces/package.json"
+  declared_licenses:
+  - "Apache-2.0"
+  declared_licenses_processed:
+    spdx_expression: "Apache-2.0"
+  vcs:
+    type: ""
+    url: ""
+    revision: ""
+    path: ""
+  vcs_processed:
+    type: "Git"
+    url: "<REPLACE_URL_PROCESSED>"
+    revision: "<REPLACE_REVISION>"
+    path: "plugins/package-managers/node/src/funTest/assets/projects/synthetic/yarn/workspaces"
+  homepage_url: ""
+  scopes: []
```

Comment:
Continuing on my earlier question, what do you think about having a follow-up PR that creates a scope called "workspace" which *does* list the projects that belong to a workspace as `PROJECT_DYNAMIC` dependencies?

Reply:
I'd tend to not do that, however, similar to my answer above, I'd propose to defer this decision.